protocol/pushnotificationclient/client.go

```diff
@@ -287,15 +314,47 @@ func (c *Client) Reregister(contactIDs []*ecdsa.PublicKey, mutedChatIDs []string
 	return c.Register(c.deviceToken, c.apnTopic, c.tokenType, contactIDs, mutedChatIDs)
 }
 
+// pickDefaultServesr picks n servers at random
+func (c *Client) pickDefaultServers(servers []*ecdsa.PublicKey) []*ecdsa.PublicKey {
+	// shuffle and pick n at random
+	shuffledServers := make([]*ecdsa.PublicKey, len(servers))
+	copy(shuffledServers, c.config.DefaultServers)
+	mrand.Shuffle(len(shuffledServers), func(i, j int) {
+		shuffledServers[i], shuffledServers[j] = shuffledServers[j], shuffledServers[i]
+	})
```

Comment:
`mrand` doesn't seem to have its seed set. This will default to the same seed each time.

Reply:
ah didn't know, good point!

Reply:
It's a quirk of go, in go the random libraries aren't random by default.